Feature ajs add campuses to send connection report job

ajs1037 opened the pull request:
Added Chandler, North Phoenix and Verrado to the send connection report job

jhawkzz commented on church.ccv.Utility/Jobs/SendConnectionReport.cs:
```diff
@@ -302,6 +308,81 @@ public void Execute( IJobExecutionContext context )
+            if (chandlerEmailList.Any())
+            {
+                var mergeFields = new Dictionary<string, object>();
+                mergeFields.Add("Title", "Surprise Campus");
```
This needs to match the campus name

jhawkzz commented on church.ccv.Utility/Jobs/SendConnectionReport.cs:
```diff
@@ -452,6 +533,81 @@ public void Execute( IJobExecutionContext context )
+            if (northphoenixEmailList.Any())
+            {
+                var mergeFields = new Dictionary<string, object>();
+                mergeFields.Add("Title", "Surprise Campus");
```
This needs to match the campus name

jhawkzz commented on church.ccv.Utility/Jobs/SendConnectionReport.cs:
```diff
@@ -677,6 +833,81 @@ public void Execute( IJobExecutionContext context )
+            if (verradoEmailList.Any())
+            {
+                var mergeFields = new Dictionary<string, object>();
+                mergeFields.Add("Title", "Surprise Campus");
```
Campus name

ajs1037 commented:
@jhawkzz I moved the function outside of the execute. You told me to merge as soon as I did that, but I would like you to check my code one last time. Moving the function outside of the execute function wasn't too straightforward and I just want to make sure I did everything correctly. I complied and made sure everything ran. No errors. I'm sure!! Thank you so much for your help on this. 


Here is a summary of what I did:


1. Moved SendCampusEmailList out of the Execute function
2. Had to add a rockContext parameter to SendCampusEmailList
3. Added the rockContext parameter to every SendCampusEmailList function call inside execute.

Current errors at this point:
    Severity	Code	Description	Project	File	Line	Suppression State
    Error	CS0103	The name 'sevenDaysAgo' does not exist in the current context	
    Error	CS0103	The name 'sevenDaysAgo' does not exist in the current context	
    Error	CS0103	The name 'thirtyDaysAgo' does not exist in the current context
    Error	CS0103	The name 'thirtyDaysAgo' does not exist in the current context
    Error	CS0103	The name 'thirtyDaysAgo' does not exist in the current context
    Error	CS0103	The name 'thirtyDaysAgo' does not exist in the current context
    Error	CS0103	The name 'connectionPageId' does not exist in the current context
    Error	CS0103	The name 'systemEmail' does not exist in the current context
    Error	CS0103	The name 'systemEmail' does not exist in the current context
    Error	CS0103	The name 'systemEmail' does not exist in the current context
    Error	CS0103	The name 'systemEmail' does not exist in the current context
    Error	CS0103	The name 'context' does not exist in the current context

4. Moved time frames and get system email code from Execute to SendCampusEmailList
        // timeframes
        var sevenDaysAgo = RockDateTime.Now.AddDays( -7 );
        var thirtyDaysAgo = RockDateTime.Now.AddDays( -30 );

        // get system email
        SystemEmail systemEmail = null;
        if ( systemEmailGuid.HasValue )
        {
            SystemEmailService emailService = new SystemEmailService( rockContext );
            systemEmail = emailService.Get( systemEmailGuid.Value );
        }

5. Had to add systemEmailGuid and connectionPageId as parameters to the SendCampusEmailList funtion.
6. Moved  context.Result = "Emails sent.";
